docs/service_spec.md

```diff
@@ -47,6 +47,14 @@ This means that the SDK supports Big Segments and can be configured with a custo
 
 For tests that involve Big Segments, the test harness will provide parameters in the `bigSegments` property of the configuration object, including a `callbackUri` that points to one of the test harness's callback services (see [Callback endpoints](#callback-endpoints)). The test service should configure the SDK with its own implementation of a Big Segment store, where every method of the store delegates to a corresponding endpoint in the callback service.
 
+#### Capability `"service-endpoints"`
+
+This means that the SDK supports setting the base URIs for the streaming, polling, and events services separately from whether those services are enabled.
```

Comment:
Note that this doesn't require for the SDK API to have a thing that is actually called "service endpoints". All that matters is that the base URI properties/setters are independent of whether the service is enabled.

So for instance, this is true in the C SDKs because there is an independent `LDConfigSetEventsURI` function. And it's true in most of the other SDKs, but in some cases (like Java) when we started moving toward component-scoped configuration with nested builders, we ended up making the base URI properties only available when the service is enabled. We are now walking back from that because it made things a bit inconvenient for people who use Relay Proxy and/or a private LD instance.